templates/outline.yaml

```diff
@@ -202,9 +202,7 @@ spec:
       http:
         paths:
           - path: /
-            pathType: Prefix
+#            pathType: Prefix
             backend:
-              service:
-                name: {{ .Values.name }}
-                port:
-                  name: web
+              serviceName: {{ .Values.name }}
```

Comment:
Syntax isn't correct for current stable ingress api version, see old format which was correct